Header header
string source_vehicle_id
int32 object_id
string object_class
float32 confidence
float32 x_min
float32 y_min
float32 x_max
float32 y_max